docker-compose-goerli.yml

```diff
@@ -11,14 +11,14 @@ x-defaults: &defaults
       max-file: "50"
 
 services:
-  agent1:
+  optimism-to-arbitrum:
```

Comment:
Might be nice to name the container so that it is clear this is on goerli, e.g. `goerli-optimism-to-arbitrum`.